controllers/create_collectordeployment.go

```diff
@@ -80,7 +80,7 @@ func (r *SearchReconciler) CollectorDeployment(instance *searchv1alpha1.Search)
 	}
 	err := controllerutil.SetControllerReference(instance, deployment, r.Scheme)
 	if err != nil {
-		log.V(2).Info("Could not set control for search-collector deployment", err)
+		log.V(2).Info("Could not set control for search-collector deployment")
```

Comment:
Should this be logged as error so that we can see the error?
log.Error(err, "Could not set control for search-collector deployment")

Reply:
We can skip logging the error .